### PR TITLE
Update Plotly.js to 1.53.0

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,8 +8,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+
+### Updated
+- [#784](https://github.com/plotly/dash-core-components/pull/784)
+  - [Feature release 1.53.0](https://github.com/plotly/plotly.js/releases/tag/v1.53.0) which contains:
+    - `rangebreaks` on date axes [#4614](https://github.com/plotly/plotly.js/pull/4614)
+    - (x|y) unified `hovermode` [#4620](https://github.com/plotly/plotly.js/pull/4620)
+    - "hovered data" mode to `spikesnap` [#4665](https://github.com/plotly/plotly.js/pull/4665)
+    - "full-json" export format to `Plotly.toImage` and `Plotly.dowloadImage` [#4593](https://github.com/plotly/plotly.js/pull/4593)
+    - node.customdata and link.customdata in `sankey` traces [#4621](https://github.com/plotly/plotly.js/pull/4621)
+    - `opacityscale` for `surface` traces [#4480](https://github.com/plotly/plotly.js/pull/4480)
```
Comment (Marc-Andre-Rivet) [inline CHANGELOG.md:22]: List of added features. If some specific bugs should be included, let me know or please add them :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,8 +8,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 - [#776](https://github.com/plotly/dash-core-components/pull/776) Update dcc.Link to set href as children if children not defined. Makes href a required prop as well.
 - [#767](https://github.com/plotly/dash-core-components/pull/767) Updated dcc.Link to respond to click modifiers, and added a target prop.
-- [#774](https://github.com/plotly/dash-core-components/pull/774) Fixed dcc.Location firing callbacks for wrong property. 
-- [772](https://github.com/plotly/dash-core-components/pull/772) Modified dcc.Link to work with absolute paths if refresh=True. 
+- [#774](https://github.com/plotly/dash-core-components/pull/774) Fixed dcc.Location firing callbacks for wrong property.
+- [772](https://github.com/plotly/dash-core-components/pull/772) Modified dcc.Link to work with absolute paths if refresh=True.
+
+### Updated
+- [#784](https://github.com/plotly/dash-core-components/pull/784)
+  - [Feature release 1.53.0](https://github.com/plotly/plotly.js/releases/tag/v1.53.0) which contains:
```
Comment (nicolaskruchten) [inline CHANGELOG.md:16]:
```suggestion
  - [Feature release of Plotly.js 1.53.0](https://github.com/plotly/plotly.js/releases/tag/v1.53.0) which contains:
```

Comment (nicolaskruchten) [reply CHANGELOG.md:16]: Maybe even add "which powers the `Graph` component :)

**Comment by nicolaskruchten** [comment]
💃

**Pull request merged**